internal/server/handlers.go

```diff
@@ -298,8 +298,13 @@ func (a *API) Version(c *gin.Context, r *api.EmptyRequest) (*api.Version, error)
 	return &api.Version{Version: internal.FullVersion()}, nil
 }
 
-// UpdateIdentityInfoFromProvider calls the identity provider used to authenticate this user session to update their current information
-func (a *API) UpdateIdentityInfoFromProvider(rCtx access.RequestContext) error {
+// providerUserUpdateThreshold is the duration of time that must pass before a
+// users session is attempted to be validated again with an external identity provider.
+// This prevents hitting IDP rate limits.
+const providerUserUpdateThreshold = 5 * time.Minute
```

Comment:
5 minutes seems pretty frequent. Could we cache this for longer? Maybe a couple hours?

Reply:
Changed to 2 hours